### DISCUSSION TITLE
Misses done in easier parts of a map should reduce PP by a lower amount

### DISCUSSION THREAD

**Discussion by StalkyBoi43815** [ppy/osu#27762] [Ideas]
When you get a lot of 300s in an easy part of the map, you get a low amount of PP. When you get a lot of 300s in a hard part of the map, you get a lot more PP. However, when you miss (or drop sliders), PP is reduced by the same amount no matter how easy or hard that section of the map is. I find this unfair, and I think it's illogical to not include this change. From a user's perspective, this change only benefits us; and developers and contributors will have a more balanced scoring system to possibly use elsewhere. There is no harm in this change I can easily see. This would only benefit players who suffer from inconsistency.

**Comment by TextAdventurer12** [comment]
PP doesn't have judgement location, and almost certainly never will. The information for this isn't stored for scores. 300s have the same effect no matter where you get them in the map

**Reply by StalkyBoi43815** [reply]
> PP doesn't have judgement location, and almost certainly never will. The information for this isn't stored for scores. 300s have the same effect no matter where you get them in the map

That's what I find to be unfair. Players suffer from inconsistency and they can do little to change this. So you are saying that scores set before the implementation of this change will be unaffected?

**Reply by Rekunan** [reply]
> Players suffer from inconsistency

If you're looking for an "inconsistency buff", there's the [combo scaling removal rework](https://github.com/ppy/osu/pull/16280).

> they can do little to change this

I believe that almost anyone could change this, assuming that they have no mental or physical illness of any kind. Though prob only thru a lot of work, most likely involving some massive mental reframing.

> scores set before ... will be unaffected?

If this change were to ever happen, then yes because it's impossible to have that data without just reading the replay files, which has a whole other set of issues (only top-1k per map has replays, insane resources needed to process everything, etc.)

Disregarding the logistics, I disagree with the proposal as it implies that the player would've FC'd or SS'd the map without the notes they got non-300's on, which is mathematically incorrect.
